specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,214 @@
+                  "required":true,
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationRequest"
+                  }
+               }
+            ]
+         }
+      }
+   },
+   "definitions":{  
+      "SubscriptionCreationResult":{  
+         "properties":{  
+            "subscriptionId":{  
+               "type":"string"
+            }
+         }
+      },
+      "SubscriptionCreationRequest":{  
+         "properties":{  
+            "subscriptionDisplayName":{  
+               "type":"string"
+            },
+            "Owners":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/AdPrincipal"
+               },
+               "description":"An array of Active Directory Principals"
+            },
+            "offerId":{  
+               "type":"string"
+            }
+         }
+      },
+      "ErrorResponse":{  
+         "description":"Describes the format of Error response.",
+         "type":"object",
+         "properties":{  
+            "code":{  
+               "description":"Error code",
+               "type":"string"
+            },
+            "message":{  
+               "description":"Error message indicating why the operation failed.",
+               "type":"string"
+            }
+         }
+      },
+      "AdPrincipal":{  
+         "description":"Active Directory Principal for subscription creation delegated permission",
+         "properties":{  
+            "tenantId":{  
+               "description":"Tenant id of the Principal",
+               "type":"string"
+            },
+            "objectId":{  
+               "description":"Object id of the Principal",
+               "type":"string"
+            }
+         }
+      },
+      "OperationListResult":{  
+         "description":"status of the subscription POST operation.",
+         "properties":{  
+            "pendingOperations":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/SubscriptionOperation"
+               }
+            }
+         }
+      },
+      "SubscriptionOperation":{  
+         "description":"status of the subscription POST operation.",
+         "properties":{  
+            "Location":{  
+               "description":"GET this URL to retrieve the status of the asynchronous operation.",
+               "type":"string"
+            },
+            "Retry-After":{  
+               "description":"The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+               "type":"string"
+            }
+         }
+      }
+   },
+   "parameters":{  
+      "apiVersionParameter":{  
+         "name":"api-version",
+         "in":"query",
+         "required":true,
+         "type":"string",
+         "description":"Version of the API to be used with the client request. Current version is 2015-06-01"
+      }
+   }
```

Comment:
I believe we need to add this

Reply:
"security": [
    {
      "azure_auth": [
        "user_impersonation"
      ]
    }
  ],
  "securityDefinitions": {
    "azure_auth": {
      "type": "oauth2",
      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
      "flow": "implicit",
      "description": "Azure Active Directory OAuth2 Flow",
      "scopes": {
        "user_impersonation": "impersonate your user account"
      }
    }
  }